### PR TITLE
Mimekitty removal

### PR DESCRIPTION
**Pull request by Groudonmaster**
Refer to issue #2009

Intent of your Pull Request

Removes Mimekitty at Shadow's request. Replaces it with the honkbot because it's better and I miss it.

Changelog

:cl:
rscdel: Mimekitty has been slaughtered for breaking its vow of silence
rscadd: To replace the late Mimekitty we're bluespacing in a honkbot. Because they're better anyways.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
https://github.com/yogstation13/yogstation/pull/2218

**Comment by ShadowDeath6** [comment]
congrats on fixing your errors

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 1]
```diff
@@ -260,44 +260,4 @@
 	..()
 	if(L.a_intent == "harm" && L.reagents && !stat)
 		L.reagents.add_reagent("nutriment", 0.4)
-		L.reagents.add_reagent("vitamin", 0.4)
```
Comment (ShadowDeath6) [inline code/modules/mob/living/simple_animal/friendly/cat.dm]: I assume this isn't intended

**Comment by ShadowDeath6** [comment]
"it's" should be "its" in the cl

**Comment by Groudonmaster** [comment]
I blame autocorrect

**Comment by Mrcreepssss** [comment]
RIP mimekitty.

**Pull request merged**